nextcloud/Containerfile

```
@@ -0,0 +1,53 @@
+FROM dir://base-buster
```

```suggestion
FROM registry.mafiasi.de/base-buster
```
would probably be better

---

Fixed